lib/ProMotion/table/cell/table_view_cell_module.rb

```diff
@@ -163,9 +163,7 @@ def map_content_mode_symbol(symbol)
 
     def map_selection_style_symbol(symbol)
       {
-        none:     UITableViewCellSelectionStyleNone,
-        blue:     UITableViewCellSelectionStyleBlue,
```

Comment:
Good point. For what it's worth, may be worth updating the documentation to make this clearer. Have updated the PR.